docs/administrator-documentation/moderne-platform/how-to-guides/organizations-service.md

```diff
@@ -11,12 +11,25 @@ Let's walk through everything you need to know to build and integrate such a ser
 
 ## Integration requirements
 
+This can be done by either providing the required files on the Agent service or by setting up an organization service. 
+Running an Organization service allows for more flexibility when setting up the required GraphQL and REST endpoint. While 
+providing the requisite files on the Agent requires less setup. 
+
+Note: When selecting which option is best for you, keep in mind that you can change the approach you have taken later. 
+
+## Providing the files on the Agent service
+When providing files on the Agent service, 
+
+1. Create and deploy an Organizations service that fulfills [this GraphQL contract](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/resources/schema/organizations.graphqls)  and [this REST contract](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/java/io/moderne/organizations/OrganizationController.java) your environment_
+2. [Configure your Moderne agent to point to this service](./agent-configuration/configure-agent-file-service.md)
```

Comment:
These instructions don't make sense to me. This section is about providing files to the Agent -- so users wouldn't need to set up a service, right? I think this may just be an accidental copy/paste.

Reply:
yes you are correct, IDK what I was thinking